packages/foam-cli/src/commands/hello.ts

```diff
@@ -23,7 +23,7 @@ hello world from ./src/hello.ts!
     const {args, flags} = this.parse(Hello)
     const name = flags.name ?? 'world'
 
-    const wm = new WorkspaceManager('./foo');
+    const wm = new NoteGraph();
```

Comment:
would make sense to call this "ng" - or maybe just "g" :)

Reply:
Or, gosh, even `graph` to be painfully explicit :)

(This is in the dummy code in CLI, however, so not mega important, since this file won't live through first real commit to the CLI project)

Reply:
good point, here I was just fixing the object used, wasn't paying attention to the name of the variable

as it's just a dummy I will leave it that way

Reply:
nevermind, there was a compilation error afterwards, so I just commented this part of the function :)